arbitrum-client/src/abi.rs

```diff
@@ -22,6 +23,26 @@ abigen!(
     ]"#
 );
 
+#[cfg(feature = "integration")]
```

Comment:
nit: Can we add a comment here delineating the difference in the generated ABI, it's simpler than trying to compare them manually if I'm reading this for the first time